_posts/en/2021-12-8-throwback-to-nantes-2021-devfest-my-first-conference.md

```diff
@@ -0,0 +1,169 @@
+
+The main features of React Query are front-end/back-end synchronization, REST/GraphQL fetching, the query state management, cache management and cached data update. To enjoy these features, the library give us many hooks (`useQuery`, `useMutation` and `useQueryClient`).
+
+Advanced features are also available, like paginated queries, “See more”, infinite scroll, query canceling, etc...
+
+Olivier had the time to show us main features with source code and tell us about advanced ones in 15 minutes! Hats off to him, because in this short amount of time, he made me want to use it in my next side project...
+
+### Next.js to the rescue of my front-end
+
+_By Nordwin Hoff_
+
+Next.js slowly becomes the future of the fast front-end development giving React a Node.js server that allow it doing some server side rendering.
+
+Nordwin does a little reminder of what is Next.js and what does it bring to a project (he talks about rendering modes, lazy loading, integrated component, etc...) before he explains how he added it to an existing codebase.
+
+I will not give you details as it’s very specific. That being said, his methodology to show us Next.js features worked well because it’s applied to a real case to which it’s easy to identify.
+
+For me that does not know Next.js, it was a very interesting talk. To make it even better, Nordwin is a very fluent speaker.
+
+### Vue 3 and its ecosystem
+
+_By Nicolas Firzzarin_
+
+I’ve always been interested in Vue.js but never went through it. I did the “Hello world” of the first versions, but I never took the time to study advanced concepts of Vue, and that’s why I attended this talk.
+
+Vue 3 seems to be inspired from some React concepts to optimize the developer experience keeping its strength: performance.
+
+The Composition API, reactive refs, proxy handlers, Suspense API, asynchronous components, libraries, etc... Everything is covered! Nicolas serves us all its knowledge on a plate, his talk is very consistent. The only issue for me was that he was comparing Vue 2 and Vue 3, but my lack of knowledge on Vue 2 did not allow me to observe the improvement made on Vue 3. A little reminder about Vue 2 would have been welcomed to really help me appreciate the talk that seemed to me very complete!
+
+### Art & entropy: Chaos in your front-end
+
+_By Thibaud Courtoison_
+
+I have to admit, the title of the talk attracts me, but means nothing to me. What will Thibaud talk about? I don’t know. Yet, I will attend his talk.
+
+What a great decision! I discover chaos engineering, as he explains the origins (Netflix, 2011), which consists - to be short - of breaking things to look at the behaviour of the infrastructure, the application, etc...
```

Comment:
```suggestion
What a great decision! I discover chaos engineering, as he explains the origins (Netflix, 2011), which consists - to be short - of breaking things to look at the behavior of the infrastructure, the application, etc...
```

Reply:
"behavior" is the american way to say "behaviour" 🙂